doc/nuklear.html

```diff
@@ -2258,6 +2307,9 @@
    - [x]: Major version with API and library breaking changes
    - [yy]: Minor version with non-breaking API and library changes
    - [zz]: Bug fix version with no direct changes to API
+- 2019/06/23 (4.01.0) - Added nk_***_get_scroll and nk_***_set_scroll for groups, windows, and popups
+- 2019/06/12 (4.00.3) - Fix panel background drawing bug
```

Comment:
I also got these lines after rebasing. Are those correct?

Reply:
I believe they reflect the changes in CHANGELOG

Reply:
Ah, that's a generated file, my bad.